### ISSUE TITLE
Make everything available at the beginning of the research project

### ISSUE THREAD

**Issue by gwaygenomics** [carpenterlab/open-science-rules#4]
This includes code, results, data, documentation, discussions, analysis decisions, etc.

I've found that this one make people the most uncomfortable. Especially with regards to the data. It is definitely understandable to be weary of being scooped or other nefarious acts. It is not easy to let go control over hard collected data that (probably) costs a lot of 💸 to acquire. It is definitely easy for me, as a [research parasite](https://doi.org/10.1056/NEJMe1516564), to ask for data to be public so this rule will be tricky to craft.

## Benefits

* Real time tracking of decisions
* assigning credit to researchers
* how updated data and analyses alters results
* links together code with discussion/analysis decisions
* enables others to participate (new collaborators can easily join)
* Big time benefit for early career investigators to be able to point to specific scientific contributions that are not yet included as formal publications (or even as preprints yet)

What other things should be discussed in this rule? I can think of some other themes (just brainstorming)

* Public funding should provide public data
* Everyone is too busy with their own stuff to care about scooping you
* Incentive structures (promotions, reputation, prestige, etc.) are not really aligned with this practice? (discussed a bit in #2 )

**Comment by allaway** [comment]
I agree with the sentiment, but should discuss the balance of this with things like data QC (eg. don't want to put bad data out there), or data governance (protection of patient rights means some data simply cannot be made public).

**Comment by gwaygenomics** [comment]
> (protection of patient rights means some data simply cannot be made public)

I think this warrants its own rule (and subsequent discussion). See #12

**Comment by allaway** [comment]
I agree, thanks for making a "rule" out of this!
